### ISSUE TITLE
remove conv. pp's state column 

### ISSUE THREAD

**Issue by nesnoj** [reegis/reegis#6]
The state column should be removed, as the state is available in the conv. pp dataset only.
To my mind, it is better not to keep it and allow the user to add it by using powerplants.add_regions_to_powerplants() instead to get a fully populated state column later.

cf. 20f31af0947b525afe3c084d45191c99743d0703

**Comment by uvchik** [comment]
You are right in the long run. I need it no, but I will keep it in mind and remove it within the next months.
